cli/bin/test-binary.sh

```diff
@@ -5,10 +5,11 @@ BASEDIR=`dirname $0`/..
 cd $BASEDIR
 
 PATH=$(pwd)/dist:$PATH
-cp tests/data/dcos.toml $DCOS_CONFIG
```

Comment:
Is this because `cp` doesn't exist in Windows? I forget why we required to copy the file. I think before we modifying `test/data/dcos.toml`/`$DCOS_CONFIG` in the TC build. I think now we are updating `/etc/hosts`. If we are not doing that anymore then maybe we can remove this `cp`.